pakku-lock.json

```diff
@@ -13078,27 +13114,27 @@
                 },
                 {
                     "type": "modrinth",
-                    "file_name": "TerraFirmaGreg-Core-Modern-0.6.1.jar",
+                    "file_name": "TFG-Core-Modern-0.6.0.jar",
                     "mc_versions": [
                         "1.20.1"
                     ],
                     "loaders": [
                         "forge"
                     ],
                     "release_type": "release",
-                    "url": "https://cdn.modrinth.com/data/lNttW2Xl/versions/V5J04AzG/TerraFirmaGreg-Core-Modern-0.6.1.jar",
-                    "id": "V5J04AzG",
+                    "url": "https://cdn.modrinth.com/data/lNttW2Xl/versions/NInE2PwT/TFG-Core-Modern-0.6.0.jar",
```

Comment:
Is utilizing the 0.6.0 version of TFG Core intentional? latest is 0.6.1

Reply:
fixed!